### DISCUSSION TITLE
Translation Plug-in

### DISCUSSION THREAD

**Discussion by FelipeRearden** [Taitava/obsidian-shellcommands#92] [Offtopic]
I think I found something that you gonna like :)

https://github.com/twentytwokhz/language-translator
 
:)

**Comment by Taitava** [comment]
Huh, you really found it withing an hour of it's initial commit in GitHub! That was fast! 🚀

I have to try it next time I'm doing Swedish exercises. One thing I noticed from the readme is that the _target language_ needs to be written as a prefix to what will be translated. So it's a bit different than what I'm looking for (defining the target language in settings, or perhaps multiple target languages). But if I like the plugin, I can make a feature request.

Thank you for tipping me off! 🙂

**Reply by FelipeRearden** [reply]
Yeah, it's a new plugin! 

I hope this plugin helps you with your Swedish learning :)

Have a great day!
